modules/get-started/pages/whats-new.adoc

```diff
@@ -6,6 +6,28 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/blob/main/CHANGELOG.md[Redpanda Connect changelog^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+
+== Version 4.68.0
+
+link:https://github.com/redpanda-data/connect/releases/tag/v4.68.0[See the full release notes^].
+
+=== Component updates
+
+This release adds the following new components:
+
+* Processors:
+** xref:components:processors/jira.adoc[`jira`] (experimental): Executes Jira API queries based on input messages and returns structured results. The processor handles pagination, retries, and field expansion automatically.
+
+
+This release adds support for the following new fields:
+
+* Inputs components
+** xref:components:inputs/redpanda.adoc['redpanda']: xref:components:inputs/redpanda.adoc#unordered_processing['unordered_processing'] - Configures partition consumers to allow parallel and therefore unordered processing of messages of any given partition. This allows for better utilization of processing threads and asynchronous publishing at the output level. The maximum parallelization of each partition is determined by the checkpoint_limit field.
```

Comment:
```suggestion
** xref:components:inputs/redpanda.adoc[`redpanda`]: ***xref:components:inputs/redpanda.adoc#unordered_processing[`unordered_processing`] - Configures partition consumers to allow parallel and therefore unordered processing of messages of any given partition. This allows for better utilization of processing threads and asynchronous publishing at the output level. The maximum parallelization of each partition is determined by the `checkpoint_limit` field.
```
Also update the description if you accept the suggestion for [modules/components/partials/fields/inputs/redpanda.adoc](https://github.com/redpanda-data/rp-connect-docs/pull/324/files#diff-095764454a330f1e643111df0ef3dd5be8156e9aec23c7cce90f34b97d1de227)